base_edifact/models/edifact.py

```diff
@@ -199,14 +199,15 @@ def map2odoo_product(self, seg, pia=None):
             ['5', ['1276', 'SA', '', '9']]
         SA. Supplier's Article Number
         """
+        res = {}
+        # Fallback on SA if no EAN given
```

Comment:
```suggestion
        # Set default for code based on SA if given
```
Hi Tri, imo, maybe need to change this comment

Reply:
Could you add [this one](https://github.com/OCA/edi/blob/9cd45f58ace990702b5a816f166e51943850f46a/base_business_document_import/models/business_document_import.py#L693) to your PR's description to make explicit that: the matching has already supported in case having both `code and barcode` also, it will be helpful for reviewer on OCA side